docs/best_practices/time_series.rst

```diff
@@ -157,3 +157,15 @@ Unknown Resolution
 If the ``resolution`` of a :ref:`nwb-schema:sec-TimeSeries` is unknown, use ``-1.0`` or ``NaN`` to indicate this.
 
 Check function: :py::meth:`~nwbinspector.checks.time_series.check_resolution`
+
+
+
+.. _best_practice_non_zero_rate:
+
+Zero Rate
+~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
Zero Rate
~~~~~~~~~
```

length of tildes should match length of title; sphinx build will warn about this but I don't think it causes an error yet